pkgs/development/tools/jo/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, fetchurl}:
+let
+  s = # Generated upstream information
+  rec {
+    baseName="jo";
+    version="0.9";
+    name="${baseName}-${version}";
+    url="https://github.com/jpmens/jo/releases/download/v${version}/${baseName}-${version}.tar.gz";
+    sha256="07r4m9g8696gvgz9z9m2zk83nc1ifa5siiyqplp6ijg5d5sd2mz7";
+  };
+  buildInputs = [];
+in
+stdenv.mkDerivation {
+  inherit (s) name version;
+  inherit buildInputs;
+  src = fetchurl {
+    inherit (s) url sha256;
+  };
+
+  enableParallelBuilding = true;
+
+  meta = {
+    inherit (s) version;
+    description = ''A small utility to create JSON objects'';
+    license = stdenv.lib.licenses.gpl2Plus;
+    maintainers = [stdenv.lib.maintainers.markus1189];
+    platforms = stdenv.lib.platforms.linux ++ stdenv.lib.platforms.darwin;
```

Comment:
I'm pretty sure you can do `platforms.unix` or even `platforms.all` here, seeing as they provide packages for OpenBSD as well. Can't see anything but standard C/POSIX includes.